pkg/controller/cleanup/handle_cleanup.go

```diff
@@ -266,7 +267,27 @@ func (c *Controller) HandleCleanup() http.Handler {
 		func() {
 			defer enobs.RecordLatency(ctx, time.Now(), mLatencyMs, &result, &item)
 			item = tag.Upsert(itemTagKey, "UNCLAIMED_USER_REPORTS")
-			if count, err := c.db.PurgeUnclaimedUserReports(c.config.UserReportUnclaimedMaxAge); err != nil {
+
+			realms, _, err := c.db.ListRealms(pagination.UnlimitedResults)
```

Comment:
Instead of looping over all the realms, can we just `SELECT MAX(user_report_unclaimed_max_age)`?

Reply:
Sorry, I meant `MAX(realm.short_code_max_minutes)`

Reply:
updated